lib/autoproj/cli/main_ci.rb

```diff
@@ -15,22 +31,21 @@ class MainCI < StandaloneCI
             option :report, type: "string", default: "cache-pull.json",
                             desc: "a file which describes what has been pulled"
```

Comment:
"report generated by the cache-pull command, usually cache-pull.json"

Reply:
I would also rename the option to use_pull_report for clarity

Reply:
Problem is that until we completely remove "auto cache pull" from `ci build`, the `--report` option is both input and output. Input if no `--cache` is given or output otherwise.

Reply:
What if someone does `autoproj ci build --cache cache --use_pull_report cache-pull.json` ? Also, I think `--report` is much better and consistent with the other commands (ci osdeps, ci cache-pull and ci cache-push all use `--report`). Since `--cache` is going to be removed (hopefully) soon, isn't the temporary slight ambiguity a reasonable compromise for the sake of consistency?